src/app/components/layout/search-bar/search-bar.component.ts

```diff
@@ -24,21 +26,68 @@ export class SearchBarComponent implements OnDestroy {
    * If true, it is not possible to start another search and the UI is shown busy.
    */
   searching = false;
+  /**
+   * Indicates if the node synchronization alert msg must be shown.
+   */
+  showSyncWarning = false;
 
+  private nodeUrlSubscription: Subscription;
+  private syncSubscription: Subscription;
   private operationSubscription: Subscription;
 
   constructor(
     public searchService: SearchService,
     private router: Router,
     private translate: TranslateService,
-  ) { }
+    private apiService: ApiService,
+  ) {
+    // Each time the node URL is changed, check if the node is in sync.
+    this.nodeUrlSubscription = this.apiService.localNodeUrl.subscribe(() => {
+      this.checkSyncState(0);
+    });
+  }
 
   ngOnDestroy() {
+    if (this.nodeUrlSubscription) {
+      this.nodeUrlSubscription.unsubscribe();
+    }
+    if (this.syncSubscription) {
+      this.syncSubscription.unsubscribe();
+    }
     if (this.operationSubscription && !this.operationSubscription.closed) {
       this.operationSubscription.unsubscribe();
     }
   }
 
+  /**
+   * Checks if the node is in sync. If it is not, the procedure will be automatically
+   * repeated after a delay.
+   * @param delayMs Delay before performing the actual check.
+   */
+  private checkSyncState(delayMs: number) {
+    if (this.syncSubscription) {
+      this.syncSubscription.unsubscribe();
+    }
+
+    // Get the data.
+    this.syncSubscription = of(0).pipe(delay(delayMs), mergeMap(() => this.apiService.getSyncState())).subscribe(response => {
```

Comment:
Codacy has a fix for the issue: [Expected parentheses around arrow function argument.](https://app.codacy.com/gh/skycoin/skycoin-explorer/pullRequest?prid=7163626)
```suggestion
    this.syncSubscription = of(0).pipe(delay(delayMs), mergeMap(() => this.apiService.getSyncState())).subscribe((response) => {
```